### ISSUE TITLE
feature request: ability to cancel text input

### ISSUE THREAD

**Issue by df-0** [nvrmind/IssueTracker#84]
once the keyboard's up (e.g. if you pressed the save as button) there doesn't seem to be any way to cancel it (for example if you didn't need to press the save as button). Not the end of the world, but kinda feels a bit weird that you can't do it!

**Comment by daseyb** [comment]
If you press the right trigger the keyboard should go away again :) I do
agree that that's not obvious at all though!

**Comment by df-0** [comment]
oh, cool - i tried the buttons that usually toggle/cancel the main panel/layer panel, since they usually seem to be what closes other windows as well, didn't try that!

**Comment by daseyb** [comment]
Yep, it should be the same button for the keyboard, gonna change that!
